### ISSUE TITLE
Internal Server Error 500 

### ISSUE THREAD

**Issue by pfremm** [kubernetes/dashboard#1998]
Dashboard version: 1.6.1
Kubernetes version: 1.6.2
Operating system: Centos 7.3

Reproduce Steps:
 Deploy cluster and add kubernetes dashboard deployment and have SELinux enabled.  The logs only show this with no additional information on next steps.

```invalid configuration: no configuration has been provided
[restful] recover from panic situation: - runtime error: invalid memory address or nil pointer dereference
    /home/travis/.gimme/versions/go1.7.1.linux.amd64/src/runtime/asm_amd64.s:479
    /home/travis/.gimme/versions/go1.7.1.linux.amd64/src/runtime/panic.go:458
    /home/travis/.gimme/versions/go1.7.1.linux.amd64/src/runtime/panic.go:62
    /home/travis/.gimme/versions/go1.7.1.linux.amd64/src/runtime/sigpanic_unix.go:24
    /home/travis/build/kubernetes/dashboard/.tmp/backend/src/github.com/kubernetes/dashboard/vendor/k8s.io/client-go/kubernetes/clientset.go:376
    /home/travis/build/kubernetes/dashboard/.tmp/backend/src/github.com/kubernetes/dashboard/src/app/backend/handler/apihandler.go:171
    /home/travis/build/kubernetes/dashboard/.tmp/backend/src/github.com/kubernetes/dashboard/vendor/github.com/emicklei/go-restful/filter.go:19
    /home/travis/build/kubernetes/dashboard/.tmp/backend/src/github.com/kubernetes/dashboard/src/app/backend/handler/apihandler.go:140
    /home/travis/build/kubernetes/dashboard/.tmp/backend/src/github.com/kubernetes/dashboard/vendor/github.com/emicklei/go-restful/filter.go:19
    /home/travis/build/kubernetes/dashboard/.tmp/backend/src/github.com/kubernetes/dashboard/src/app/backend/handler/apihandler.go:103
    /home/travis/build/kubernetes/dashboard/.tmp/backend/src/github.com/kubernetes/dashboard/vendor/github.com/emicklei/go-restful/filter.go:19
    /home/travis/build/kubernetes/dashboard/.tmp/backend/src/github.com/kubernetes/dashboard/src/app/backend/handler/apihandler.go:204
    /home/travis/build/kubernetes/dashboard/.tmp/backend/src/github.com/kubernetes/dashboard/vendor/github.com/emicklei/go-restful/filter.go:19
    /home/travis/build/kubernetes/dashboard/.tmp/backend/src/github.com/kubernetes/dashboard/vendor/github.com/emicklei/go-restful/container.go:269
    /home/travis/build/kubernetes/dashboard/.tmp/backend/src/github.com/kubernetes/dashboard/vendor/github.com/emicklei/go-restful/container.go:120
    /home/travis/.gimme/versions/go1.7.1.linux.amd64/src/net/http/server.go:1726
    /home/travis/.gimme/versions/go1.7.1.linux.amd64/src/net/http/server.go:2022
    /home/travis/build/kubernetes/dashboard/.tmp/backend/src/github.com/kubernetes/dashboard/vendor/github.com/emicklei/go-restful/container.go:287
    /home/travis/.gimme/versions/go1.7.1.linux.amd64/src/net/http/server.go:2022
    /home/travis/.gimme/versions/go1.7.1.linux.amd64/src/net/http/server.go:2202
    /home/travis/.gimme/versions/go1.7.1.linux.amd64/src/net/http/server.go:1579
    /home/travis/.gimme/versions/go1.7.1.linux.amd64/src/runtime/asm_amd64.s:2086```

**Comment by floreks** [comment]
I think I know what is the cause of it. Could you check logs for more information? There should be additional error printed before the panic. The issue is that client is generated dynamically and if there is an error during configuring client we print error and set response but do not return from function.

We will fix that but anyway this is not your main issue. I'm sure there is additional error thrown that prevents client from being created.

**Comment by pfremm** [comment]
It does seem to be resolved on docker 1.13.1.  I rebuilt my cluster with that and it seems to be working at the moment so I don't have any further logs.
